.github/workflows/ci.yml

```diff
@@ -5,7 +5,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: ['10', '11', '12', '13', '14']
+        node: ['12', '13', '14']
```

Comment:
this reflects a breaking change that we'll need to make sure is captured in `engines.node` as well